### PR TITLE
added in timer test, removed warning messages with commandline flag

### PR DESCRIPTION
**Pull request by dhuang612**
removed exception warnings
```
 [chrome-exception] { timestamp: 1573797858767.935,
[1]   exceptionDetails:
[1]    { exceptionId: 2,
[1]      text: 'Uncaught',
[1]      lineNumber: 361,
[1]      columnNumber: 0,
[1]      url:
```

using command line flag
```
--ignore-exceptions
```
added to the test-browser script.

added in one additional test to make sure that the timer counted down.

separated tests into two describes

one is to check for existence of the functions

the second runs the functions to confirm they are running correctly.

### CODE REVIEW

File: tests/webspec.js [hunk 1]
```diff
@@ -1,8 +1,9 @@
 'use strict'
 const expect = chai.expect;
 
+describe('Game Function Tests', function(){
```
Comment (Arcaster42) [inline tests/webspec.js]: Any reason you switched out of ES6 syntax?

Comment (dhuang612) [reply tests/webspec.js]: I can change the top one back to using arrow functions.

for the second describe I needed it for setting the time out
https://mochajs.org/#arrow-functions
Passing arrow functions (aka "lambdas") to Mocha is discouraged. Lambdas lexically bind this and cannot access the Mocha context. For example, the following code will fail:

**Comment by dhuang612** [comment]
ok standardized it to not use es6 for access to this for the second describe.
Please let me know if anything else needs to be done

**Comment by Arcaster42** [comment]
Typically ES6 is preferred syntax except in some cases that specifically warrant using ES5 (such as when you want to avoid binding `this`). But I'll merge this for now. Thanks for your work!

**Pull request merged**